docs/_layouts/pages.html

```diff
@@ -2,25 +2,106 @@
 layout: default
 class: fill-light
 options: full
+examples: 
+  - name: Getting started
+    id: get-started
+    level: h3
+  - name: Add a style
+    id: start-styles
+    level: h4
+    class: icon paint quiet
+  - name: Add a layer
+    id: start-data
+    level: h4
+    class: icon tileset quiet
+  - name: Dynamic Styling
+    id: data-visualization
+    level: h3
+  - name: Style points
+    id: point
+    level: h4
+    class: icon marker quiet
+  - name: Style lines
+    id: line
+    level: h4
+    class: icon line quiet
+  - name: Style polygons
+    id: polygon
+    level: h4
+    class: icon polygon quiet 
+  - name: User Interaction
+    id: user-interaction
+    level: h3
+  - name: Popups
+    id: popup
+    level: h4
+    class: icon contact quiet
+  - name: Filter
+    id: filter
+    level: h4
+    class: icon layers quiet
+  - name: Animations
+    id: animation
+    level: h3
+  - name: Plugins
+    level: h3
+  - name: Geocoding
+    id: mapbox-geocoding
+    level: h4
+    class: icon search quiet
+  - name: Directions
+    id: mapbox-directions
+    level: h4
+    class: icon street quiet
+  - name: GL Draw
+    id: gl-draw
+    level: h4
+    class: icon pencil quiet
+  - name: Change map options
+    id: change-option
+    level: h3
+  - name: Misc
+    id: misc
+    level: h3
```

Comment:
Did some renaming of categories and rearranging of examples. Take a look and let me know what you think!